ci/test.sh

```diff
@@ -10,8 +10,8 @@ chown -R ubuntu:ubuntu /workspace/
 chown -R ubuntu:ubuntu /cache/
 
 # Add DNS entries for proxy protocol tests
-echo '127.0.1.11 stargate.local' >> /etc/hosts
-echo '127.0.1.12 stargate.local' >> /etc/hosts
+echo '127.0.1.11 internal-stargate.local' >> /etc/hosts
```

Comment:
shouldn't we retain the previous entries as well? Could you elaborate more on this?

Reply:
Our CI environment uses local IP addresses, so `ProxyProtocolQueryInterceptor` will resolve the peers using the "internal" DNS name.